mixins/_hyphens.scss

```diff
@@ -0,0 +1,11 @@
+//
+// layout helpers
+// --------------------
+//
+//
+
+@mixin hyphens {
+    hyphens: auto;
+    -webkit-hyphens: auto;
+    -moz-hyphens: auto;
+}
```

Comment:
since https://github.com/postcss/autoprefixer/issues/61 it seems to work in postcss, since we switched to it now ;).

@lhtdesignde please remove this mixin